### PR TITLE
Show interval value on click

### PR DESCRIPTION
**Pull request by ahuang11**
Adds CSS classes to the buttons + display value on speed increase

https://github.com/user-attachments/assets/121147f1-cf4a-4b9a-83c0-ef7a0929542e

Closes https://github.com/holoviz/panel/issues/5207

### CODE REVIEW

**Comment by philippjfr** [comment]
Just checking, but would showing `fps` rather than `ms` be clearer?

**Comment by ahuang11** [comment]
I like that idea of using fps.

File: panel/models/player.ts [hunk 1]
```diff
@@ -320,8 +333,17 @@ export class PlayerView extends WidgetView {
     this.set_frame(this.model.end)
   }
 
+  updateSpeedButton(button: HTMLButtonElement, interval: number, originalSVG: string): void {
+    const fps = 1000 / interval
+    button.innerHTML = `${fps.toFixed(1)}<br>fps`
+    setTimeout(() => {
+      button.innerHTML = originalSVG
+    }, 1500) // Show for 1.5 seconds
```
Comment (hoxbro) [inline panel/models/player.ts:341]: I suggest having a parameter for this so it can be set in Python.

File: panel/models/player.ts [hunk 2]
```diff
@@ -320,8 +333,17 @@ export class PlayerView extends WidgetView {
     this.set_frame(this.model.end)
   }
 
+  updateSpeedButton(button: HTMLButtonElement, interval: number, originalSVG: string): void {
+    const fps = 1000 / interval
+    button.innerHTML = `${fps.toFixed(1)}<br>fps`
+    setTimeout(() => {
+      button.innerHTML = originalSVG
+    }, this.model.preview_duration) // Show for 1.5 seconds
```
Comment (hoxbro) [inline panel/models/player.ts:341]:
```suggestion
    }, this.model.preview_duration)
```

How does this behave if `preview_duration` is equal to zero.

File: panel/widgets/player.py [hunk 3]
```diff
@@ -31,6 +33,10 @@ class PlayerBase(Widget):
         default='once', objects=['once', 'loop', 'reflect'], doc="""
         Policy used when player hits last frame""")
 
+    preview_duration = param.Integer(default=1500, doc="""
```
Comment (hoxbro) [inline panel/widgets/player.py:36]:
```suggestion
    preview_duration = param.Integer(default=1500, bounds=(0, None), doc="""
```

**Review by philippjfr — APPROVED** [review]

**Pull request merged**